deltachat-ios/DC/events.swift

```diff
@@ -48,9 +48,9 @@ public func callbackSwift(event: CInt, data1: CUnsignedLong, data2: CUnsignedLon
             if done {
                 UserDefaults.standard.set(true, forKey: Constants.Keys.deltachatUserProvidedCredentialsKey)
                 UserDefaults.standard.synchronize()
-                let appDelegate = UIApplication.shared.delegate as! AppDelegate
-                // appDelegate.appCoordinator?.showTab(index: 3)
-                AppDelegate.lastErrorDuringConfig = nil
+                if let appDelegate = UIApplication.shared.delegate as? AppDelegate {
```

Comment:
@r10s you probably can leave out the cast/if let line here since appDelegate is not used inside the if let block (AppDelegate.lastErrorDuringConfig only accesses the static variable lastErrorDuringConfig https://github.com/deltachat/deltachat-ios/blob/f15345b46159f4a268da649e9e39930c34d76a8b/deltachat-ios/AppDelegate.swift#L24 )

Reply:
right, i was too deep into the as-pattern now seeing that the statement is void at all ;)